src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -0,0 +1,24 @@
+import { Routes } from '@angular/router';
+import { UserRouteAccessService } from 'app/core/auth/user-route-access-service';
+import { ExamManagementComponent } from 'app/exam/manage/exam-management.component';
+
+export const examManagementRoute: Routes = [
+    {
+        path: ':courseId/exams',
```

Comment:
Maybe I get it wrong but the route is already defined in the `courseManagementRoute`. `path: ''` should be enough here right?